cishouseholds/pipeline/survey_responses_version_digital_ETL.py

```diff
@@ -12,6 +13,232 @@ def digital_specific_transformations(df):
+            [
+                "Self-employed",
+                [
+                    "Self-employed",
+                    None,
+                    "Not looking for paid work. This includes looking after the home or family or not wanting a job or being long-term sick or disabled",  # noqa: E501
+                    None,
+                ],
+            ],
+            ["Self-employed", [None, None, None]],
+            ["Not working (unemployed, retired, long-term sick etc.)", ["Self-employed", None, None, None]],
+            [
+                "Not working (unemployed, retired, long-term sick etc.)",
+                [
+                    "Not in paid work. This includes being unemployed or doing voluntary work",
+                    None,
+                    "Looking for paid work and able to start",
+                    None,
+                ],
+            ],
+            [
+                "Not working (unemployed, retired, long-term sick etc.)",
+                [
+                    "Not in paid work. This includes being unemployed or doing voluntary work",
+                    None,
+                    "Not looking for paid work. This includes looking after the home or family or not wanting a job or being long-term sick or disabled",  # noqa: E501
+                    None,
+                ],
+            ],
+            [
+                "Not working (unemployed, retired, long-term sick etc.)",
+                ["Not in paid work. This includes being unemployed or doing voluntary work", None, "Retired", None],
+            ],
+            [
+                "Not working (unemployed, retired, long-term sick etc.)",
+                ["Not in paid work. This includes being unemployed or doing voluntary work", None, None, None],
+            ],
+            [
+                "Student",
+                [
+                    "Education",
+                    None,
+                    None,
+                    "A child below school age and not attending a nursery or pre-school or childminder",
+                ],
+            ],
+            [
+                "Student",
+                [
+                    "Education",
+                    None,
+                    None,
+                    "A child below school age and attending a nursery or pre-school or childminder",
+                ],
+            ],
+            ["Student", ["Education", None, None, "A child aged 4 or over at school"]],
+            ["Student", ["Education", None, None, "A child aged 4 or over at home-school"]],
+            [
+                "Student",
+                [
+                    "Education",
+                    None,
+                    None,
+                    "Attending a college or other further education provider including apprenticeships",
+                ],
+            ],
+        ],
+        column_list,
+    )
+    df = assign_column_value_from_multiple_column_map(
+        df,
+        "is_self_isolating_detailed",
```

Comment:
This was renamed in 2229
```suggestion
        "self_isolating_reason",
```